components/Stars.tsx

```diff
@@ -5,18 +5,18 @@ import starStyles from '../styles/Stars.module.css'
 export type StarsProps = {
   defaultOptionIndex?: number
   number?: number
-  title?: string
   updateCallback?: (update: number) => void
 }
 
-const Star = ({ selected }: { selected?: boolean }) => {
+const Star = ({ index, selected }: { index?: string; selected?: boolean }) => {
   let fill = 'rgba(30, 30, 30, 0.3)'
   if (selected) {
     fill = 'rgba(255, 214, 0, 0.8)'
   }
 
   return (
     <svg
+      aria-label={index ? `${index + 1} stars` : ''}
```

Comment:
I tried both this approach and using a ref to refer to the id, but since the id of each star element is not simply the index number, it wound up adding a lot of code! I THINK passing the index might be more efficient but happy to further talk about this!